lighthouse-core/report/html/renderer/details-renderer.js

```diff
@@ -75,8 +75,11 @@ class DetailsRenderer {
    */
   _renderBytes(details) {
     // TODO: handle displayUnit once we have something other than 'kb'
-    const value = Util.i18n.formatBytesToKB(details.value, details.granularity);
-    return this._renderText(value);
+    // Note that 'kb' is historical and actually represents KiB.
+    const value = Util.i18n.formatBytesToKiB(details.value, details.granularity);
+    const text = this._renderText(value);
+    text.setAttribute('title', Util.i18n.formatBytes(details.value));
```

Comment:
nit:
```suggestion
    textEl.title = Util.i18n.formatBytes(details.value);
```
is slightly better here